docs/mkdocs/docs/api/basic_json/emplace_back.md

```diff
@@ -13,6 +13,12 @@ Creates a JSON value from the passed parameters `args` to the end of the JSON va
 `Args`
 :   compatible types to create a `basic_json` object
 
+## Iterator invalidation
+
+By adding an element to an array, a reallocation can happen, in which case all iterators (including
```

Comment:
...to the end of an array... (since it's different if added in a different place, which is a different function)